helpers/utils/util.ts

```diff
@@ -38,3 +38,11 @@ export const accurateWPM = (
    const wordsTyped = typedEntries / 5;
    return Math.max((wordsTyped - errorCount) / minutes, 0);
 };
+
+export const accuratePercentage = (
```

Comment:
We are looking at how accurately you type a character and not a word...so the accuracy should be based on correct and wrong characters instead of words.